packages/plugin-sample-api/src/graphql/resolvers/index.ts

```
@@ -0,0 +1,47 @@
+import customScalars from '@erxes/api-utils/src/customScalars';
+import { sendCoreMessage } from "../../messageBroker";
+import mutations from './mutations';
+import queries from './queries';
+import { IContext } from '../../connectionResolver';
+import { IUser } from '@erxes/api-utils/src/types';
+
+const Sample = {
+  async currentType(sample, { }, { models }: IContext) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)